doc/team.rst

```diff
@@ -0,0 +1,112 @@
+.. title:: Team
+
+PyGMT Team Gallery
+==================
+
+We are an international team dedicated to building a Pythonic API for the
+Generic Mapping Tools (GMT). Our goal is to improve GMT's accessibility for
+new and experienced users by creating user-friendly interfaces with the GMT
+C API, supporting rich display in Jupyter notebooks, and integrating with
+the PyData ecosystem.
+
+All our welcome to become involved with the PyGMT project! For more information
+about how to get involved, see the :doc:`contributing`.
+
+Distinguished Contributors
+--------------------------
+
+PyGMT Distinguished Contributors are recognized for their substantial
```

Comment:
See [gmt/AUTHORS.md](https://github.com/GenericMappingTools/gmt/blob/master/AUTHORS.md):
> ## The Founders
> GMT was conceived and designed in the late 1980s and early 1990s by Paul Wessel and Walter H.F. Smith. The > two founders built and maintained GMT from version 1 through 3.

I have another thought. How about adding a short description (or a single category)  for PyGMT founders? That's a great way to show respect to them. I know **Leonardo** is the first to start the PyGMT project, and **Dongdong** and **weiji14** soon joined the project and continue to contribute to the project. We could have more teams and more active contributors in the future, and re-organize the team gallery, but the founders won't change.

Reply:
Leo is mentioned as the founder at https://www.pygmt.org/dev/overview.html. I'm not really a founder, more like an early adopter. We could put a 'Founder' label/tag under Leo's name I suppose.

Reply:
OK, so then I should probably also add Paul with a 'Founder' tag, since I think him and Leo were both involved with getting the funding to start the project which is a substantial contribution. Sound OK?

Reply:
> OK, so then I should probably also add Paul with a 'Founder' tag, since I think him and Leo were both involved with getting the funding to start the project which is a substantial contribution. Sound OK?

Sound good. But it's better to have more comments/thoughts from @weiji14 and @seisman since they might know more about the history.

Reply:
Most of what I know about PyGMT's early history (or gmt-python as it was back then) is from Leo's Scipy 2017 talk (https://www.youtube.com/watch?v=93M4How7R24). I'm ok with adding Paul in.

Reply:
Done in https://github.com/GenericMappingTools/pygmt/pull/1308/commits/d0e22aa1370c35cd142910cca1aa3443f8aa7813